src/components/Nav/Nav.js

```diff
@@ -16,7 +16,7 @@ import { getComponentNamespace } from '../../globals/namespace';
 
 export const navNamespace = getComponentNamespace('nav');
 
-const { forEach, map } = Children;
```

Comment:
This abstraction was confusing for me in context further down because `forEach` and `map` are language features. I think it would help to remove this.